README.md

```diff
@@ -17,9 +17,13 @@ Dienstensysteem v2
      `pip install -U python-ldap --global-option=build_ext --global-option="-I$(xcrun --show-sdk-path)/usr/include/sasl"`
 8. Create database (`createdb dienst2`, `createuser dienst2`)
 9. Initialise database [using manage.py](https://www.jetbrains.com/pycharm/help/running-tasks-of-manage-py-utility.html) (`manage.py migrate`)
-10. Install bower dependencies (`bower install`)
-11. Start server
-12. You should be good to go!
+10. Initialize user auth (choose what's applicable)
+    * Connect to [CH VPN](https://ch.tudelft.nl/vpn/) and log in with your CH account. Or,
+    * Connect to your own LDAP server (Change `AUTH_LDAP*` values in `dienst2/settings.py`). Or,
```

Comment:
Would it be an idea to put the needed variables (maybe commented out) in the example template?